test/labeling/test_analysis.py

```diff
@@ -0,0 +1,172 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+import scipy.sparse as sparse
+
+from snorkel.labeling.analysis import (
+    confusion_matrix,
+    error_buckets,
+    label_conflict,
+    label_coverage,
+    label_overlap,
+    lf_conflicts,
+    lf_coverages,
+    lf_empirical_accuracies,
+    lf_overlaps,
+    lf_polarities,
+    lf_summary,
+    single_lf_summary,
+)
+
+L = [
+    [0, 0, 1, 0, 0, 1],
+    [0, 0, 0, 3, 0, 0],
+    [3, 0, 0, 0, 0, 1],
+    [2, 0, 3, 0, 1, 1],
+    [0, 0, 0, 0, 0, 0],
+    [2, 0, 1, 3, 2, 1],
+]
+
+Y = [1, 2, 3, 1, 2, 3]
+
+
+class TestAnalysis(unittest.TestCase):
+    def setUp(self) -> None:
+        self.L = sparse.csr_matrix(np.array(L))
+        self.Y = np.array(Y)
+
+    def test_label_coverage(self) -> None:
+        self.assertEqual(label_coverage(self.L), 5 / 6)
+
+    def test_label_overlap(self) -> None:
+        self.assertEqual(label_overlap(self.L), 4 / 6)
+
+    def test_label_conflict(self) -> None:
+        self.assertEqual(label_conflict(self.L), 3 / 6)
+
+    def test_lf_polarities(self) -> None:
+        polarities = lf_polarities(self.L)
+        self.assertEqual(polarities, [[2, 3], [], [1, 3], 3, [1, 2], 1])
```

Comment:
lf_polarities should return a consistent type--probably always a list?

Reply:
https://github.com/HazyResearch/snorkel/pull/1168/files#diff-5839d69014187bdf1a2f60ccee964363R73

Reply:
Right, I see that the code _allows_ for both right now. But I think we should change it so that it only returns a List of Lists.

Reply:
Ah I see, you're saying the lone ints are unexpected. I agree. Can change now.

Reply:
Or add a flag?

Reply:
The flag would only be applicable some of the time, since multi-polar LFs would still need to be a collection of some sort. I say just make it always a list of lists.

Reply:
Cool on it

Reply:
Will do in a stacked PR so as not to complicate rebases. Not really in scope for this diff either.